src/pages/ProjectStart/Components/CardItem.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+function CardItem(props) {
+  const { title, redTitle, buttonText } = props;
+  return (
+    <CardItemWrapper>
```

Comment:
```suggestion
    <Container>
```
- 가장 바깥 쪽에 있는 친구한테는 Container라고 고민 없이 일단 이름 주는 편인데 어떨까요?
- CardItem 컴포넌트라는 사실은 이미 알고 있기 때문에 CardItem이라는 단어는 생략해도 좋아보입니다.

Reply:
동의합니다~!수정했습니다